README.md

```diff
@@ -63,6 +63,22 @@ The reconciliation process will verify the below steps:
 1. It will look for `ScopeTemplate` that `ScopeInstance` is referencing. if it is not referencing then throw an error with the appropriate message.
 2. If it is referencing and if the `namespaces` array is empty, a single `ClusterRoleBinding` will be created. Otherwise, a `RoleBinding` will be created in each of the `namespaces`. These resources will include an owner reference to the `ScopeInstance` CR.
 
+## Installation
+To install the latest release of `oria-operator`, run:
+```
+kubectl apply -f https://github.com/operator-framework/oria-operator/releases/latest/download/oria-operator.yaml
```

Comment:
Getting `not found` for this after release it will generate right?

Reply:
Yeah it should be generated once a release is created. We currently don't have any releases so it won't work for now.

Reply:
If you want to see that it works properly you can replace `operator-framework` with `everettraven` as I created a test release on my fork: https://github.com/everettraven/oria-operator/releases/tag/v0.0.3

Reply:
got it. I will try these instructions afterwards. Looks good to me.